spec/models/note_spec.rb

```diff
@@ -1,5 +1,121 @@
 require 'rails_helper'
 
+RSpec.shared_context 'with note setup' do |utility_type, _note_type, word_count|
+  let(:utility) { create(:utility, type: utility_type.to_s) }
+  let(:content) { Faker::Lorem.sentence(word_count: word_count) }
+end
+
+RSpec.shared_examples 'a valid note' do |utility_type, note_type, word_count|
+  include_context 'with note setup', utility_type, note_type, word_count
+
+  it "creates a valid #{note_type} note for #{utility_type} with #{word_count} words" do
+    note = create(:note, content: content, note_type: note_type, utility: utility)
+    expect(note).not_to be nil
+  end
+end
+
+RSpec.shared_examples 'an invalid note' do |utility_type, note_type, word_count|
+  include_context 'with note setup', utility_type, note_type, word_count
+
+  it "throws error when creating an invalid #{note_type} for #{utility_type} with #{word_count} words" do
+    expect { create(:note, content: content, note_type: note_type, utility: utility) }.to raise_error(ActiveRecord::RecordInvalid)
+  end
+end
+
 RSpec.describe Note, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject(:note) do
+    create(:note)
+  end
```

Comment:
```suggestion
  subject(:note) { create(:note) }
```
Si el largo nos da para meterlo en una sola línea, bienvenido sea :hugs: